corehq/sql_db/util.py

```diff
@@ -2,15 +2,22 @@
 from __future__ import unicode_literals
 import uuid
 from collections import defaultdict
+from numpy import random
 
-from corehq.sql_db.config import partition_config
 from django.conf import settings
 from django import db
 from django.db.utils import InterfaceError as DjangoInterfaceError
 from functools import wraps
 from psycopg2._psycopg import InterfaceError as Psycopg2InterfaceError
 import six
-from corehq.sql_db.models import PartitionedModel
+from memoized import memoized
+
+from corehq.sql_db.config import partition_config
+from corehq.util.quickcache import quickcache
+
+
+ACCEPTABLE_STANDBY_DELAY_SECONDS = 3
```

Comment:
Can we make this configurable via localsettings. Also wondering if it shouldn't be per DB since some DB's may be more tollerant than others (thinking of UCR vs main DB).

Reply:
Sure, I will make it a localsetting.

It will be easiest to make it minimum of all DBs, but if we need it to be DB specific, should we let this be configurable under `LOAD_BALANCED_APPS` per app, per db, assuming that the acceptable delay would be different per app and per db or under something like 
 `settings.STANDBY_ACCEPTABLE_DELAYS`

```
STANDBY_ACCEPTABLE_DELAYS = {
'ucrstandby': 3,
'pgmainstandby': 5
}
```

Or we could also add another key (like 'ACCEPTABLE_DELAY` to settings.DATABASES dict

Reply:
Yea, I think adding it to `settings.DATABASES` would be good

Reply:
Done. Also companion ansible PR https://github.com/dimagi/commcare-cloud/pull/1677